app/scripts/components/reconciliation-reports/reconciliation-report.js

```diff
@@ -121,37 +198,75 @@ class ReconciliationReport extends React.Component {
         </section>
 
         <section className='page__section'>
-          <div className='heading__wrapper--border'>
-            <h2 className='heading--medium with-description'>Reconciliation report</h2>
+          <div className='page__section--small'>
+            <div className='heading__wrapper--border'>
+              <h2 className='heading--medium with-description'>Reconciliation report</h2>
+            </div>
+            <Metadata data={report} accessors={reportMetaAccessors} />
           </div>
-          {!record || (record.inflight && !record.data) ? <Loading /> : <Metadata data={record.data} accessors={metaAccessors} />}
-        </section>
 
-        <section className='page__section'>
-          <div className='heading__wrapper--border'>
-            <h2 className='heading--medium heading--shared-content with-description'>
-              Files only in DynamoDB ({filesInDynamoDb.length})
-            </h2>
-          </div>
-          <SortableTable
+          <ReportTable
             data={filesInDynamoDb}
-            header={tableHeaderDynamoDb}
-            row={tableRowDyanmoDb}
-            props={['granuleId', 'filename', 'bucket', 'link']}
+            title={'Files only in DynamoDB'}
+            tableHeader={tableHeaderFiles}
+            tableRow={tableRowFile}
+            tableProps={tablePropsFile}
           />
-        </section>
 
-        <section className='page__section'>
-          <div className='heading__wrapper--border'>
-            <h2 className='heading--medium heading--shared-content with-description'>
-              Files only in S3 ({filesInS3.length})
-            </h2>
-          </div>
-          <SortableTable
+          <ReportTable
             data={filesInS3}
-            header={tableHeaderS3}
-            row={tableRowS3}
-            props={['filename', 'bucket', 'link']}
+            title={'Files only in S3'}
+            tableHeader={tableHeaderS3Files}
+            tableRow={tableRowS3File}
+            tableProps={tablePropsS3File}
+          />
+
+          <ReportTable
+            data={filesOnlyInCumulus}
+            title={'Files only in Cumulus'}
```

Comment:
Maybe change to `Granule Files only in Cumulus` and `Granule Files only in CMR` to distinguish it from the cumulus internal file report.